Editor/TestTools/EditorImguiTester/EmptyTestStrategy.cs

```diff
@@ -8,6 +8,8 @@ namespace Innoactive.CreatorEditor.TestTools
     /// </summary>
     internal class EmptyTestStrategy : IEditingStrategy
     {
+        public ICourse CurrentCourse { get; }
```

Comment:
Shouldn't this be set in the `HandleNewCourseWindow`?

Reply:
Reading the xml documentation, I would say it should do nothing.